app/views/menus/show.html.haml

```diff
@@ -1,6 +1,11 @@
 %section.container
 %h1= t('menu', day: @facade.menu_day_name)
-%h2= t('first_course')
-%h2= t('main_course')
-%h2= t('drinks')
-пше
\ No newline at end of file
+= form_with url: "/orders", method: :post do |f|
```

Comment:
This is the orders create form, so it makes sense to have `menus/:id/orders/new` in your path.